cli/src/cli.rs

```diff
@@ -405,6 +406,11 @@ pub fn parse_command(matches: &ArgMatches<'_>) -> Result<CliCommandInfo, Box<dyn
             let timestamp_pubkey = value_of(&matches, "timestamp_pubkey");
             let witnesses = values_of(&matches, "witness");
             let cancelable = matches.is_present("cancelable");
+            let sign_only = if matches.is_present("sign_only") {
```

Comment:
How is that different than `let sign_only = value_of(&matches, "sign_only");`?

Reply:
Heh, yeah, same :-)